add_licenses.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# This script adds license headers that are missing from go files
+
+
+if ! command -v addlicense 2> /dev/null
+then
+  echo "error addlicense must be installed for this rule: go get -u github.com/google/addlicense" && exit 1
```

Comment:
Let's have it say `go install ...` instead of `go get ...`

Reply:
good catch, I forgot to update that script too